lib/resolvers/error.js

```diff
@@ -76,7 +76,7 @@ const _log = error => {
     delete error2log.stack
   }
 
-  error2log = normalizeError(error2log, { locale: '' }, false)
+  error2log = normalizeError(error2log, { locale: '', get(){} }, false)
```

Comment:
Maybe something like this would be more robust to guard against similar issues in the future?

```suggestion
  const noLocaleReq = Object.assign(_clone(req), { locale: '' })
  error2log = normalizeError(error2log, noLocaleReq, false)
```

(also need to pass `req` as function arg)

Reply:
We should be more robust yes, but that is also a bad / expensive way. 
Better would be `{__proto__:req, locale:'' }`
But even better, and the only good way, imho, is to not do such mocking at all. 

As I detected that we do such bad habit things also in other places, most in tests, we anyways should discuss in a larger round → @sjvans @johannes-vogel: please put that on the list ... 

But for the time being please merge and publish this quick fix, as we are blocked by that on other urgent activities.

Reply:
BTW: ad `_clone(...)` please also stop doing and using our home-brewed mediocre deep clone implementations but always use the [standard `structuredClone()` function](https://developer.mozilla.org/en-US/docs/Web/API/Window/structuredClone) instead.